edsdme/blocks/announcements-preview/announcements-preview.js

```diff
@@ -68,6 +68,9 @@ async function fetchData(blockData, newestCards) {
     document.dispatchEvent(cardsEvent);
 
     if (apiData?.cards) {
+      if (window.location.hostname === 'partners.adobe.com') {
```

Comment:
can you use the same way of checking for prod as in partner cards?
`const isProd = prodHosts.includes(window.location.host);`